vagrant/bootstrap.sh

```diff
@@ -72,6 +72,8 @@ instance_name = "$INSTANCE_NAME"
 instance_article_dative = "der"
 # The sender address of the emails that are sent by ff-node-monitor:
 email_from = "$EMAIL_FROM"
+# Define a threshhold under which the map is treated as offline and no warning mails are sent.
```

Comment:
```suggestion
# When fewer nodes than this are online, the map is treated as offline and no warning mails are sent.
```
But I also don't understand what you mean by "the map" being offline... maybe try to rephrase that?